### PR TITLE
Feature: update pycc 4

### PR DESCRIPTION
**Pull request by maxhutch**
This is based on #24 by @Setogit , but needs to be opened within the repository in order to access the testing API key.

### CODE REVIEW

**Comment by Setogit** [comment]
Confirmed that this PR worked in my local environment and Travis as expected.  Closed the PR #24.  Thanks.

File: MLonCitrination.ipynb [hunk 1]
```diff
@@ -377,15 +372,17 @@
...
    ],
    "source": [
     "inputs = [{\"Chemical formula\": \"AlCu\"},]\n",
-    "resp = client.predict(\"27\", inputs)\n",
-    "prediction = resp['candidates'][0]['Density']\n",
-    "print(\"We predict the density of {} to be {} +/- {}\".format(inputs[0]['Chemical formula'], *prediction))"
+    "resp = client.models.predict(\"27\", inputs)\n",
+    "print(len(inputs), len(resp))\n",
+    "prediction = resp[0].get_value('Density')\n",
+    "print(\"We predict the density of {} to be {} +/- {}\".format(inputs[0]['Chemical formula'], prediction.value, prediction.loss))"
```
Comment (carenac) [inline MLonCitrination.ipynb:385]: Not a big deal, but do we want to be consistent with design table results here?

`round(prediction.value, 2)`

File: ExperimentalDesign.ipynb [hunk 2]
```diff
@@ -290,19 +287,18 @@
...
     "view_id = \"3904\"\n",
     "\n",
     "inputs = [{\"formula\": \"TiO2\"}]\n",
-    "resp = client.predict(view_id, inputs, method=\"scalar\")\n",
-    "prediction = resp['candidates']\n",
+    "prediction = client.models.predict(view_id, inputs, method=\"scalar\")\n",
     "\n",
     "print(inputs[0]['formula'])\n",
-    "print(\"We predict the ZT of {} (@ 300K) to be {} +/- {}\".format(inputs[0]['formula'], prediction[0]['Property ZT'][0], prediction[0]['Property ZT'][1]))\n"
+    "print(\"We predict the ZT of {} (@ 300K) to be {} +/- {}\".format(inputs[0]['formula'], prediction[0].get_value('Property ZT'), prediction[0].get_value('Property ZT')))\n"
```
Comment (carenac) [inline ExperimentalDesign.ipynb]: `prediction[0].get_value('Property ZT')`  should have a `.value`

**Review by carenac — CHANGES_REQUESTED** [review]

File: MLonCitrination.ipynb (hunk 1 above)
Comment (maxhutch) [reply MLonCitrination.ipynb:385]: The logic in the design results is a little more complicated than that; I'd suggest we leave it as is (its consistent with what it was before).

**Review by carenac — APPROVED** [review]

**Pull request merged**